### PR TITLE
Add `bulkBan` to `GuildManager`

### PR DESCRIPTION
**Pull request by MCausc78**
# Description

Implement https://github.com/discord/discord-api-docs/pull/6720
In `PartialGuild.createBan`/`PartialGuild.deleteBan` i changed `member` to `user` since discord allows banning users outside of server

## Connected issues & other potential problems

None

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Checklist:

- [x] Ran `dart analyze` or `make analyze` and fixed all issues
- [x] Ran `dart format --set-exit-if-changed -l 160 ./lib` or `make format` and fixed all issues
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

**Review by l7ssha — APPROVED** [review]

**Review by abitofevrything — APPROVED** [review]
LGTM, we'll wait for the docs PR to be merged before merging this.

File: lib/src/http/managers/guild_manager.dart
Comment (abitofevrything) [inline lib/src/http/managers/guild_manager.dart:217]:
```suggestion
      bannedUsers: parseMany(raw['banned_users'] as List, Snowflake.parse),
      failedUsers: parseMany(raw['failed_users'] as List, Snowflake.parse),
```

**Review by abitofevrything — CHANGES_REQUESTED** [review]

**Comment by abitofevrything** [comment]
I don't think the docs are going to be merged for a while, but seeing as many other libraries are implementing this we'll merge this in now.

**Review by abitofevrything — APPROVED** [review]

**Pull request merged**
